api_cluster_control_plane.go

```diff
@@ -145,8 +152,11 @@ func (a *SystemAdministrationPolicyClusterControlPlaneApiService) DeleteClusterC
 	localVarQueryParams := url.Values{}
 	localVarFormParams := url.Values{}
 
+	if localVarOptionals != nil && localVarOptionals.Cascade.IsSet() {
+		localVarQueryParams.Add("cascade", parameterToString(localVarOptionals.Cascade.Value(), ""))
+	}
 	// to determine the Content-Type header
-	localVarHttpContentTypes := []string{"application/json"}
+	localVarHttpContentTypes := []string{}
```

Comment:
Could I know why it is removed?

Reply:
I regenerated the code using swagger-code-gen, not sure why it is removed in the new generated code.  IT is delete and no request body, I suppose the content-type is not needed?